tests/testthat/test-graphics.R

```diff
@@ -1,19 +1,27 @@
 test_that("single plot is captured", {
-  ev <- evaluate(file("plot.R"))
+  ev <- evaluate("plot(1:10)")
   expect_length(ev, 2)
   expect_equal(classes(ev), c("source", "recordedplot"))
 })
 
 test_that("ggplot is captured", {
+  # TODO: figure out why this doesn't work interactively: probably because
+  # the root environment is wrong - needs to be parent of global env
```

Comment:
Same here, when running inside RStudio IDE on windows, it works ok for me. Trying to understand what is not working

Reply:
I have already mostly forgotten but I think I was surprised that I couldn't `library(ggplot2)` outside of the `evaluate()` call. But I now understand why that is (because we're not evaluating `print()` in quite the right environment), and why it's hard to fix. I have some notes I'll eventually turn into a comment.

Reply:
> I couldn't library(ggplot2) outside of the evaluate() call.

Oh ok I see! Thanks.